### PR TITLE
Update and rename manifests/j/JetBrains/GoLand/2021.1.1/JetBrains.GoL…

### PR DESCRIPTION
**Pull request by abdonmorales**
…and.yaml to manifests/j/JetBrains/GoLand/211.7142.13/JetBrains.GoLand.yaml

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15919)

### CODE REVIEW

File: manifests/j/JetBrains/GoLand/211.7142.13/JetBrains.GoLand.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
 PackageIdentifier: JetBrains.GoLand
-PackageVersion: 2021.1.1
+PackageVersion: 211.7142.13
 PackageName: GoLand
 Publisher: JetBrains
```
Comment (jedieaston) [inline manifests/j/JetBrains/GoLand/211.7142.13/JetBrains.GoLand.yaml:5]:
```suggestion
Publisher: JetBrains s.r.o.
```

File: manifests/j/JetBrains/GoLand/211.7142.13/JetBrains.GoLand.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
 PackageIdentifier: JetBrains.GoLand
-PackageVersion: 2021.1.1
+PackageVersion: 211.7142.13
 PackageName: GoLand
```
Comment (jedieaston) [inline manifests/j/JetBrains/GoLand/211.7142.13/JetBrains.GoLand.yaml:4]:
```suggestion
PackageName: GoLand 2021.1.1
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Two quick things to match the ARP table. Otherwise, LGTM!

**Comment by abdonmorales** [comment]
Go it, Ill update the rest of the packages with the naming convention for the JetBrain applications

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**